compiler/control/OMROptions.cpp

```diff
@@ -728,6 +737,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"enableNewX86PrefetchTLH",           "O\tenable new X86 TLH prefetch algorithm", SET_OPTION_BIT(TR_EnableNewX86PrefetchTLH), "F"},
    {"enableNodeGC",                      "M\tenable node recycling", SET_OPTION_BIT(TR_EnableNodeGC), "F"},
    {"enableOnsiteCacheForSuperClassTest", "O\tenable onsite cache for super class test",       SET_OPTION_BIT(TR_EnableOnsiteCacheForSuperClassTest), "F"},
+   {"enableOldEDO",                       "O\tenable the old EDO mechanism", SET_OPTION_BIT(TR_EnableOldEDO), "F", NOT_IN_SUBSET},
```

Comment:
Should this order be reversed with `enableOnsiteCacheForSuperClassTest` ?

Reply:
Indeed. Fixed in force push